package.json

```diff
@@ -62,6 +62,7 @@
     "@ministryofjustice/frontend": "3.3",
     "@ministryofjustice/hmpps-connect-dps-components": "^1.2.0",
     "@sentry/node": "^7.54.0",
+    "accessible-autocomplete": "^3.0.1",
```

Comment:
Why does it need to be installed? The way it's being used it can just be copied from https://github.com/alphagov/accessible-autocomplete/tree/main/dist